Dockerfile

```diff
@@ -1,14 +1,12 @@
 FROM molecularplayground/node-js
-RUN apk add --no-cache bash
+RUN apk add bash
```

Comment:
The apk package manager could use cache to fetch bash, I got rid of no cache so we could use this, but I was mistakened to think this would work cause Ryan informed me that the images are built from scratch. I got rid of it the line anyways cause we already had sh installed.